examples/ads.amp.html

```diff
@@ -712,17 +712,24 @@ <h2>Aniview</h2>
   </amp-ad>
 
   <h2>AppNexus with JSON based configuration multi ad</h2>
-  <amp-ad width="300" height="250"
-      type="appnexus"
-      data-target="apn_ad_1"
-      json='{"pageOpts":{"member": 958}, "adUnits": [{"disablePsa": true, "tagId": 6063968,"sizes": [300,250],"targetId": "apn_ad_1"}, {"tagId": 6063968,"sizes": [728,90],"targetId":"apn_ad_2"}]}'>
-  </amp-ad>
 
-  <amp-ad width="728" height="90"
+    <amp-ad width="728" height="90"
```

Comment:
nit: can you make indentation consistent across the three examples here?

Reply:
done, thanks